src/app/modules/main/wallet_section/accounts/model.nim

```diff
@@ -173,34 +191,71 @@ QtObject:
     of ModelRole.CanSend:
       result = newQVariant(item.canSend())
 
+  proc updateBalance*(self: Model, address: string, balance: CurrencyAmount, assetsLoading: bool) =
+    let i = self.findAccountIndex(address)
+    if i < 0:
```

Comment:
Wondering if this is an error or something worth logging. WDYT?
In theory we shouldn't try to update accounts we don't have.